plugin.py

```diff
@@ -144,6 +146,51 @@ def log_solar_charger(
         print(json.dumps(delta))
         sys.stdout.flush()
 
+    def log_dcdc_converter(
+        self,
+        bl_device: BLEDevice,
+        cfg_device: ConfiguredDevice,
+        data: DcDcConverter,
+        id_: str,
+    ):
+        delta = {
+            "updates": [
+                {
+                    "source": {
+                        "label": "Victron",
+                        "type": "Bluetooth",
+                        "src": bl_device.address,
+                    },
+                    "timestamp": datetime.datetime.utcnow().isoformat() + "Z",
+                    "values": [
+                        {
+                            "path": f"electrical.converters.{id_}.chargingMode",
```

Comment:
I don't see that (`electrical/converters/*`) in [the SignalK specification](https://github.com/SignalK/specification). I do see discussion about DC/DC convertors in the repo history, though.

Is this something we need to raise with them again, to add the key?

Reply:
Should this be `electrical.chargers` which does exist? As far as I see, the only Victron DC-DC convertor products are 3-stage battery chargers.

Reply:
> I don't see that (`electrical/converters/*`) in [the SignalK specification](https://github.com/SignalK/specification). I do see discussion about DC/DC convertors in the repo history, though.
> 
> Is this something we need to raise with them again, to add the key?

sorry for the late reply, I was away on an extended vacation and didn't really check my email.

are you on the signalk-dev slack? there have been a few discussions there about where in the path tree the dc-dc stuff might belong.

Reply:
> are you on the signalk-dev slack? there have been a few discussions there about where in the path tree the dc-dc stuff might belong.

No, I can't get into it :( https://groups.google.com/g/signalk/c/14rumtRLl4s/m/dAYoMbBkDQAJ

Reply:
Filed Signalk/Specification#652

Reply:
can we merge this PR now and deal with the SK specs later?

Reply:
Yes